valgrind/riscv64.patch

```diff
@@ -1,63 +1,48 @@
--            '1e22b75b95252583774916dab9dcbc8663495107e15dd1ddf397744b288265fcffd1f456d306bc610989a1f650ae66a8ebeb84d253be312db2ab9fdc9fe1407f')
--b2sums=('80024371b3e70521996077fba24e233097a6190477ced1b311cd41fead687dcc2511ac0ef723792488f4af08867dff3e1f474816fda09c1604b89059e31c2514'
+-            'e0cec39381cefeca09ae4794cca309dfac7c8693e6315e137e64f5c33684598726d41cfbb4edf764fe985503b13ff596184ca5fc32b159d500ec092e4cf8838c')
+-b2sums=('9312761b0531006725f13270984b26c48f71ebe66e355b04410d7c01773c9b78ec21db3259cab398e58b3b68f93f5f074db06efe1f359bd7cf423515576191a4'
 -        'SKIP'
--        'af556fdf3c02e37892bfe9afebc954cf2f1b2fa9b75c1caacfa9f3b456ebc02bf078475f9ee30079b3af5d150d41415a947c3d04235c1ea8412cf92b959c484a'
--        '5af1f467c8d22334e14e6c2878120550ce5f1e36b61f8180601b8673493a85cca8a0c3a804c281ab51fb04fe5e3706edeb897bbd5486f87d9143f472cd4c46b5'
--        '305234e118eeb389c95ca8afa45239b0386199dee6f5058a3ddca06a91b2069e1b9831ac5e31badf6d8bd0d43eef3f260d95b976a509d73284149a64d9ea0563')
-+sha512sums=('5e0634ed81970c0b9d274511e622b505798c601886f25ee86a2c81614c7aae8ff0d663e725f1a4418d8590e39ad85cc195f9015185f342e1a06608feb1e2b978'
-+            '1f69f4b472643886ed76f1c5dde77ad55bad749add67507f830fe205b4160414e4798db1af7df8b84b40bc452b31fd423f9560914c72d146c906bd704287c4d9'
-+            'e0cec39381cefeca09ae4794cca309dfac7c8693e6315e137e64f5c33684598726d41cfbb4edf764fe985503b13ff596184ca5fc32b159d500ec092e4cf8838c')
-+b2sums=('48ba35d9710cc321ad695031873089065658c7dc6f3a0f97ad79be559371fe1d8d029a3b33420780e37ff322b185884491850acee32feb24369aa793f043daa4'
-+        '613f40328c104a2b7cd303859559e9e92bbac34a6ad3cf676a14a03262d6a4c2159db4f147381932c391aacbbce8a80609c05e4c58037f68f0e1c28b01d440bd'
-+        'af556fdf3c02e37892bfe9afebc954cf2f1b2fa9b75c1caacfa9f3b456ebc02bf078475f9ee30079b3af5d150d41415a947c3d04235c1ea8412cf92b959c484a')
+-        'af556fdf3c02e37892bfe9afebc954cf2f1b2fa9b75c1caacfa9f3b456ebc02bf078475f9ee30079b3af5d150d41415a947c3d04235c1ea8412cf92b959c484a')
  options=(!lto) # https://bugs.kde.org/show_bug.cgi?id=338252
  
  prepare() {
 +  mv valgrind-riscv64-${_commit} valgrind-${pkgver}
    cd valgrind-${pkgver}
    patch -Np1 < ../valgrind-3.7.0-respect-flags.patch
    sed -i 's|sgml/docbook/xsl-stylesheets|xml/docbook/xsl-stylesheets-1.79.2-nons|' docs/Makefile.am
- 
--  # can be dropped with the next release, see
--  # https://gitlab.archlinux.org/archlinux/packaging/packages/valgrind/-/issues/2
--  patch -Np1 < ../valgrind_3_22_0_s390x-linux_memfd_secret.patch
--  patch -Np1 < ../valgrind_3_22_0_fchmodat2_syscall.patch
-+  patch -Np1 < ../fix-riscv64-fp-ite-csel.patch
- 
-   autoreconf -ifv
+@@ -118,3 +110,6 @@
  }
+ 
+ # vim: ts=2 sw=2 et:
++sha256sums=('976fb5e53c6121a29bb8421e61926911c1afc4cae8292c634135e5cbc3619fde'
++            '63a0b914fc66decf25760523eb7d7031bf08bf9e7e26f891b03b9e72c375d590'
++            '4e6be3a1799c17a6e843ab1966a3a68ac0ffa83d4b2230ce1b607518c42a31a2')
```

Comment:
I'll still prefer using the original means of checksums so those unmodified patches won't cause any surprises.

Reply:
The filename is a stupid mistake on my side...using it when a patch don't run at first to test it locally...shouldn't have made it's way to the patch...will change that. Sorry. 
The checksums have been calculated by updpkgsums. But since I could resolve it without a new patch I wonder why they are different. I'll have another look at it tomorrow.
Thx for your remarks!

Reply:
Had another look....the changed checksums result from the patch using a newer commit of the project. The patch before this PR hat the same lines changed.

Reply:
No I actually meant that the sha256sums added at the end of the PKGBUILD were not present in either Arch upstream or the original patch here.